src/zkevm_specs/state.py

```diff
@@ -388,23 +391,35 @@ def check_tx_log(row: Row, row_prev: Row):
     if row.tag() == row_prev.tag():
         if tx_id != pre_tx_id:
             assert tx_id == pre_tx_id + 1
-            assert log_id == 0
+            # in RW table, log_id starts with 1 per tx, zero log_id means no any log step executed,
```

Comment:
I have the feeling that we may have redundant constraints in the state circuit; which are checking things that are already enforced by the EVM circuit.
For example, the EVM circuit is constrained to generate lookups for TxLog with correct `log_id`.  So if the EVM circuit passes the constraints, then there's no way a TxLog can exist that doesn't start with `1`.

I think for now it's OK to have some redundant constraints, but in the future I think we need to figure out a way to analyze this situation and remove redundant constraints (with confidence that we're not allowing invalid witnesses to pass)